### ISSUE TITLE
Deploy on vercel

### ISSUE THREAD

**Issue by TalonDragon000** [TalonDragon000/Hom3Town#1]
Working on getting the app to deploy on vercel. debugging...

**Comment by TalonDragon000** [comment]
figured out that i had all my codes still on localhost. updating github with necessary files.

**Comment by TalonDragon000** [comment]
tried deploying with all the files in place now. not working yet... researching.

**Comment by leetebbs** [comment]
Hey @TalonDragon000 
you will need to add a vercel.json file to your server.
https://vercel.com/guides/using-express-with-vercel#5.-configure-your-project-for-vercel
you can see an example of how i have done it in one of my projects :
https://github.com/leetebbs/dbeats/blob/main/server/vercel.json
If you add the same file into the root of your server folder it should work.

**Comment by TalonDragon000** [comment]
> Hey @TalonDragon000 you will need to add a vercel.json file to your server. https://vercel.com/guides/using-express-with-vercel#5.-configure-your-project-for-vercel you can see an example of how i have done it in one of my projects : https://github.com/leetebbs/dbeats/blob/main/server/vercel.json If you add the same file into the root of your server folder it should work.

Thanks Tebbo! 
I ended up installing vercel dependencies to my client on my comp and that created a vercel folder that was added to my gitignore. it says it's a file that shouldn't be shared, so the site works, but I guess I won't be updating my github repo with that info. 

I was also getting confused with Vite and Socket.io etc. etc. I'm sure there is a MUCH cleaner way to deploy all this. i feel like I have too much going on haha. I don't think it auto-updates to production though so I might need to manually re-assign the ENV variable... 

I'm so confused. lol XDDD BUT it works so... fixed for now? lol
